development/build/static.js

```diff
@@ -44,6 +44,16 @@ const copyTargets = [
     pattern: `*.html`,
     dest: ``,
   },
+  {
+    src: `./node_modules/ses/dist/`,
```

Comment:
Would it be possible to import this from `runLockdown` instead? It would be nice to avoid copying this directly from the dependency. Aside from being unconventional, this this is also incompatible with package managers that don't use `node_modules` (e.g. Yarn v2).

Reply:
@Gudahtt  part of the reasoning here is to take the ses dist as is without any transformations. also possible under yarn@2, would just need to rework the import